Prac07/guitarTest.py

```diff
@@ -0,0 +1,31 @@
+from Prac07.guitar import Guitar
+
+
+def main():
+
+    i = 0
```

Comment:
The variable "i" isn't used for anything, I'm guessing it was originally meant to count the guitars?